lib/PublicShare/TemplateLoader.php

```diff
@@ -48,8 +57,14 @@ public static function register(EventDispatcherInterface $dispatcher): void {
 	 *
 	 * This method should be called when loading additional scripts for the
 	 * public share page of the server.
+	 *
+	 * @param IShare $share
 	 */
-	public static function loadTalkSidebarUi() {
+	public static function loadTalkSidebarUi(?IShare $share) {
+		if ($share && $share->getNodeType() !== FileInfo::TYPE_FILE) {
```

Comment:
Would it make sense to switch to "hide by default".
This way we can help people to update their Nextcloud, as well as not allow to load the app on folders while the owner can not load the conversation in his file list?
```suggestion
		if (!$share || $share->getNodeType() !== FileInfo::TYPE_FILE) {
```

Reply:
The Talk sidebar was loaded on folders, but it was not shown, so that was not a problem. Anyway, if you want to _help_ people to update their Nextcloud, fine by me.